corehq/apps/fixtures/upload/run_upload.py

```diff
@@ -550,7 +549,7 @@ def _process_group_ownership(di, old_data_item, group_memoizer, transaction):
         old_data_item.remove_group(group)
 
     for group_name in di.get('group', []):
-        group = group_memoizer.by_name(group_name)
+        group = group_memoizer.by_name(str(group_name))
```

Comment:
This is now obsolete (it has been removed with the old uploader), but it handled a non-string group name value. This can happen when non-string values (such as numbers) are entered in the Excel sheet. A non-string group name would cause the old uploader to crash.